neurodamus/io/sonata_config.py

```diff
@@ -277,7 +291,7 @@ def make_circuit(nodes_file, node_pop_name, population_info):
         }
 
     # Compat with BlueConfig circuit definitions
```

Comment:
shouldn't we be dropping `Compat with BlueConfig` related things?

Reply:
Done